Library/Homebrew/cask/lib/hbc/dsl.rb

```diff
@@ -114,8 +115,13 @@ def language(*args, &block)
                       }
         end
       end
+
+      if args.include?(:default)
+        # :default has to be the last language block in order to assign return value of the selected `language` block to `@language`
+        @language = @language[:block].call
+      end
```

Comment:
I'm sure there is a better place/way to do this. Could need some help here, @jawshooah and @mwean.